### PR TITLE
Raster coverage

### PR DESCRIPTION
**Pull request by pkdash**
This is a fix for raster coverage calculation.

### CODE REVIEW

**Comment by pkdash** [comment]
@hyi and @dtarb Can you review this?

**Comment by hyi** [comment]
+1

File: hs_core/testing.py [hunk 1]
```diff
@@ -284,9 +284,9 @@ def raster_metadata_extraction(self):
...
         self.assertEqual(box_coverage.value['projection'], 'WGS 84 EPSG:4326')
         self.assertEqual(box_coverage.value['units'], 'Decimal degrees')
-        self.assertEqual(box_coverage.value['northlimit'], 42.11071605314457)
+        self.assertEqual(box_coverage.value['northlimit'], 42.11270614966863)
         self.assertEqual(box_coverage.value['eastlimit'], -111.45699925047542)
-        self.assertEqual(box_coverage.value['southlimit'], 41.66417975061928)
+        self.assertEqual(box_coverage.value['southlimit'], 41.66222054591102)
         self.assertEqual(box_coverage.value['westlimit'], -111.81761887121905)
 
         # there should be 2 format elements
```
Comment (dtarb) [inline hs_core/testing.py]: These look reasonable and are what I would expect given the change in logic

**Comment by dtarb** [comment]
Qualified +1 from me.  I would like to see a test deployment.  The logic changes and box coverage comparison changes look good to me, but the other code changes are not within my skill set to comment on.

**Comment by pkdash** [comment]
@dtarb This work has been deployed on playground. I wait until you are done with testing to merge this PR.

**Comment by dtarb** [comment]
Tested.  +1.

**Pull request merged**